src/orchestrator/orchestrator.rs

```diff
@@ -427,31 +437,44 @@ impl<'a> Orchestrator<'a> {
         // Create a sender and a receiver for the root of the tree
         let (root_sender, mut root_receiver) = tokio::sync::mpsc::channel(100);
 
+        // preparation ended
+        drop(prepare_span_guard);
+        drop(prepare_span);
+
         // Make all prepared jobs into real jobs and run them
         //
         // This maps each TaskPreparation with its sender and receiver to a JobTask and calls the
         // async fn JobTask::run() to run the task.
         //
         // The JobTask::run implementation handles the rest, we just have to wait for all futures
         // to succeed.
+        let run_span = tracing::debug_span!("run");
```

Comment:
The span is entered in `task.run().instrument()` (the `instrument()` call) - well actually a sub-span is used, because we have more events that are related to this span here).